db/seeds.rb

```diff
@@ -1,12 +1,29 @@
 User.create([
   {
-    name: 'test_user3',
-    email: 'test_user3@email.com',
+    name: 'Ginnie',
+    email: 'ginniehench@gmail.com',
     password: 'secret!'
   },
   {
-    name: 'Ginnie3',
-    email: 'ginniehench3@gmail.com',
+    name: 'Ellie',
+    email: 'ellie@awesomedogs.com',
     password: 'secret!'
-  }
+  },
+])
+
+Submission.create([
+  {
+    user_id: 1,
+    text: "Plants don’t get enough credit. They move. You know this. Your houseplant salutes the sun each morning. At night, it returns to center."
```

Comment:
So true! I have to rotate my little stemmy succulent or it gets all bendy following the light.